src/main/java/Task.java

```diff
@@ -0,0 +1,34 @@
+public class Task {
```

Comment:
Perhaps the Task class can be abstracted since there will not be the situation where it is instantiated (either Todo, Deadline or Event will be instantiated and not Task itself)